.github/workflows/editorconfig.yml

```diff
@@ -3,7 +3,7 @@ name: "Checking EditorConfig"
 permissions: read-all
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
Why revert the change that we just merged in https://github.com/NixOS/nixpkgs/pull/120959?

Reply:
I don't think it really needs _target? User github token should be enough to fetch the pull request metadata.

Reply:
It isn't for the metadata, it's to avoid need to manually approve the action.

Reply:
https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/